spec/requests/api/resources_spec.rb

```diff
@@ -18,20 +18,28 @@
       parameter name: :learning_unit_id, in: :path, type: :string
       operationId 'getLearningUnitResources'
 
-      let(:learning_unit_id) { create(:learning_unit).id }
+      let(:learning_unit) { create(:learning_unit) }
+      let(:learning_unit_id) { learning_unit.id }
+      let(:resource) { create(:resource, learning_unit:) }
 
-      response '200', 'Success' do
+      response '200', 'Success', create_evaluation: true do
         schema type: :array,
                items: {
                  type: :object,
                  properties: {
                    id: { type: :integer },
                    name: { type: :string },
-                   url: { type: :string }
+                   url: { type: :string },
+                   average_evaluation: { type: :string },
+                   number_of_evaluations: { type: :integer }
                  }
                },
                required: %w[id name]
-        run_test!
+        run_test! do |response|
+          data = JSON.parse(response.body)
+          expect(data[0]['average_evaluation']).to eq('3.0')
```

Comment:
Magic number: in the 'before' block you use the default factory value, is best practice to specify in the same file the value that you will be expecting, even if the factory provides one (i.e. create(:resource_evaluation, user:, evaluation: 3) )

Reply:
Done, thx.